en/docs/design/create-api/create-a-streaming-api-from-an-asyncapi-definition.md

```diff
@@ -0,0 +1,109 @@
+# Create a Streaming API from an AsyncAPI Definition
+
+An AsyncAPI definition is a format that describes asynchronous APIs. 
+
+The following types of Streaming APIs can be created using an existing AsyncAPI definition:
+
+1. [WebSocket]({{base_path}}/design/create-api/create-streaming-api/create-a-websocket-streaming-api)
+2. [WebSub (WebHooks)]({{base_path}}/design/create-api/create-streaming-api/create-a-websub-streaming-api)
+3. [Server Sent Events (SSE)]({{base_path}}/design/create-api/create-streaming-api/create-a-sse-streaming-api)
+
+Follow the instructions below to create a Streaming API in this manner.
+
+
+1. Sign in to the WSO2 API Publisher `https://<hostname>:9443/publisher` (e.g., `https://localhost:9443/publisher`).
+
+    <html><div class="admonition note">
+      <p class="admonition-title">Note</p>
+      <p>The <b>CREATE API</b> button will only appear if the user who has signed in has the creator role permission.</p>
```

Comment:
This is not clear for the user " button will only appear if the user who has signed in has the creator role permission". 
Does this means whether the user should have "Internal/creator" role or are you saying whether the user should have "API Create" permission? Shall we be more specific here?

Reply:
I have re-used what is there for REST APIs:
https://apim.docs.wso2.com/en/latest/learn/design-api/create-api/create-a-rest-api-from-an-openapi-definition/